audio.js

```diff
@@ -1,12 +1,225 @@
+//Display audio element id
 const display = document.getElementById("display");
 
-$(".playback").click(function (e) {
-    e.preventDefault();
+//Power ON and OFF
+let toggle = document.getElementById("power-switch-input");
+display.innerHTML = 'Power Off';
 
-    var clip = $(this).next('audio').get(0);
+$('.playback').attr('disabled', 'disabled'); //audio off
 
-    display.innerHTML = clip.id;
+// console.log(typeof toggle.value);
 
-    clip.currentTime = 0;
-    clip.play();
+toggle.addEventListener('input', () => {
+    if ((toggle.value == '0' && bank.value == '0') || (toggle.value == '0' && bank.value == '1')) {
+        toggle.value = '1';
+
+        display.innerHTML = "Power On";
+
+        async function keySounds(key, id) {
+            if (event.keyCode == key) {
+                let clip = document.getElementById(id);
+                await clip;
+                if (clip == null) {
+                    // console.log('clip is null');
+                    return 0;
+                }
+                // console.log(clip);
+                // console.log(clip.id);
+                display.innerHTML = clip.id;
+
+                clip.currentTime = 0;
+
+                let playPromise = clip.play();
+
+                if (playPromise !== undefined) {
+                    playPromise.then(_ => {
+                        clip.play();
+                        // console.log('play');
+                    })
+                        .catch(error => {
+                            console.log(error);
+                        })
+                }
+            }
+        }
+
+        // console.log('bank before if ' + bank.value);
+
+        if (bank.value == '0') {
```

Comment:
Again, the bank toggle code should go in a bank element trigger rather than be coded to the power toggle. There's also some duplicate code between here (line 48) and line 84 that can be reduced.

Reply:
Line 48 to 84 was on purpose based on how the DOM listens for key presses. Without it, the keys stop playing sounds based on what order the toggles are clicked.